azurerm/provider.go

```diff
@@ -261,6 +261,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_databricks_workspace":                               resourceArmDatabricksWorkspace(),
 			"azurerm_ddos_protection_plan":                               resourceArmDDoSProtectionPlan(),
 			"azurerm_dev_test_lab":                                       resourceArmDevTestLab(),
+			"azurerm_dev_test_lab_schedule":                              resourceArmDevTestLabSchedules(),
```

Comment:
since this is a resource within DevTest (as a DevTest Lab is) - can we make this:

```suggestion
			"azurerm_dev_test_schedule":                              resourceArmDevTestLabSchedules(),
```